aws-lc-rs/src/digest/sha.rs

```diff
@@ -31,12 +36,21 @@ pub const SHA512_256_OUTPUT_LEN: usize = 256 / 8;
 /// The length of a block for SHA-512-based algorithms, in bytes.
 const SHA512_BLOCK_LEN: usize = 1024 / 8;
 
-/// SHA-1 and SHA-256 are limited to an input size of 2^64-1 bits.
+/// SHA-1, SHA-224, and SHA-256 are limited to an input size of 2^64-1 bits.
+#[allow(clippy::cast_possible_truncation)]
+const SHA1_MAX_INPUT_LEN: u64 = u64::MAX;
+
+#[allow(clippy::cast_possible_truncation)]
+const SHA224_MAX_INPUT_LEN: u64 = u64::MAX;
+
 #[allow(clippy::cast_possible_truncation)]
 const SHA256_MAX_INPUT_LEN: u64 = u64::MAX;
 
 /// SHA-384, SHA-512, and SHA-512/256 are limited to an input size of 2^128-1 bits according to the spec.
 /// u64 is more than sufficient enough for practical usecases, so we limit the input length to 2^64-1 bits.
+#[allow(clippy::cast_possible_truncation)]
+const SHA384_MAX_INPUT_LEN: u64 = u64::MAX;
+
 #[allow(clippy::cast_possible_truncation)]
 const SHA512_MAX_INPUT_LEN: u64 = u64::MAX;
```

Comment:
Could we just combine these into a single variable called `SHA2_MAX_INPUT_LEN` or `DIGEST_MAX_INPUT_LEN` and keep the comments above it?

Reply:
Yeah